core/platform/auth/firebase_auth_services_test.py

```
@@ -1652,6 +1664,129 @@ def test_get_all_external_accounts_returns_empty_list_when_no_users(
         self.assertEqual(firebase_auth_services.get_all_external_accounts(), [])
 
 
+class ExternalAccountCrudTests(FirebaseAuthServicesTestBase):
+
+    def test_create_external_account_with_all_fields(self) -> None:
+        account = auth_domain.ExternalAccount('aid', 'test@example.com', False)
```

Ruff (0.14.14)

[error] Boolean positional value in function call

(FBT003)